pkg/kubelet/pod_container_deletor.go

```diff
@@ -48,7 +48,9 @@ func newPodContainerDeletor(runtime kubecontainer.Runtime, containersToKeep int)
 	go wait.Until(func() {
 		for {
 			id := <-buffer
-			runtime.DeleteContainer(id)
+			if err := runtime.DeleteContainer(id); err != nil {
+				klog.Warningf("[pod_container_deletor] DeleteContainer returned error for (id=%v): %v", id, err)
```

Comment:
Nit: I'm not sure I see other instances of `[some_descriptor]` in this file, so for consistencies sake, we may want to remove the `[pod_container_deletor]` prefix.